frontend/stories/inputs/textField/SearchInput.stories.tsx

```diff
@@ -19,7 +19,7 @@ import { SearchInput } from '@controls/search/searchInput';
 import { useState } from 'react';
 
 export default {
-	title: 'Inputs/SearchInput',
+	title: 'Inputs/Textfield/SearchInput',
 	component: SearchInput,
 	parameters: { controls: { exclude: ['ref', 'hiddenLabel'] } },
 } as ComponentMeta<typeof SearchInput>;
```

Comment:
It doesn't seem to be possible to enter values into the Default component

Reply:
The problem was the lack of context which controls the value. Note that the controlled version was also lacking of it (it will still be after the push), hence the `clear` button will not work for it.